.pre-commit-config.yaml

```diff
@@ -5,6 +5,9 @@ repos:
     hooks:
     -   id: trailing-whitespace
     -   id: flake8
+        additional_dependencies: [
+            'flake8-print==3.1.4'
```

Comment:
We should be able to use flake8-print==5.0.0, as we are no longer supporting Python 2.7 on develop!

Note that with version 5, they changed the codes to T2* not T0*.

Reply:
yesss, sure